test/common/tcp/conn_pool_impl_test.cc

```diff
@@ -9,8 +9,9 @@
 
 #include "test/mocks/event/mocks.h"
 #include "test/mocks/network/mocks.h"
-#include "test/mocks/tcp/mocks.h"
+#include "test/mocks/tcp/mocks_nats.h"
```

Comment:
Ditto regarding a TODO to either rename this back or replace this code from the code from envoyproxy/envoy.